### ISSUE TITLE
Dafang cover works inversely in home assistant

### ISSUE THREAD

**Issue by sendorm** [EliasKotlyar/Xiaomi-Dafang-Hacks#893]
### Description

When the image is not flipped in rtsp stream, setting the cover via position works as expected.
![image](https://user-images.githubusercontent.com/29329371/51629710-57f90a00-1f59-11e9-8599-5547712ac20c.png)

Start at 50. Set the position to 20 camera turns left. Set the position to 80 camera turns right to 80.

When the image is flipped:
Start at 50. Set the camera position to 60, the camera turns left instead of right. The final position updates to 40. So instead of increasing by 10, it decreases by 10.  
Set the position to 20, camera turns left instead of right this time and the final position is updated to 60.  

The cover buttons of **up** and **down** to turn +/-100 in x axis works as expected in both flipped and normal.

**Comment by sendorm** [comment]
I have updated the code as follows. It works as intended. 
https://github.com/sendorm/Xiaomi-Dafang-Hacks/blob/master/firmware_mod/scripts/mqtt-control.sh

One caveat though. When the screen is flipped the positions are reversed. So if I wanted the camera to turn right most.  I set the position to 0 for flipped and 100 for not flipped. 
The position status should be reversed for the flipped case, which I could not yet achieve.
